ios/FluentUI/Tooltip/Tooltip.swift

```diff
@@ -54,30 +57,30 @@ open class Tooltip: NSObject, TokenizedControlInternal {
                                                       tokenSet: tokenSet)
         self.anchorView = anchorView
         guard let tooltipViewController = tooltipViewController,
-              let tooltipView = tooltipViewController.view else {
+              let tooltipView = tooltipViewController.view,
+              let hostVC = hostViewController ?? window.rootViewController else {
             return
         }
 
-        let rootViewController = window.rootViewController
-        let rootView = rootViewController?.view
-        rootViewController?.addChild(tooltipViewController)
+        let hostView = hostVC.view
+        hostVC.addChild(tooltipViewController)
         self.onTap = onTap
         self.dismissMode = UIAccessibility.isVoiceOverRunning ? .tapOnTooltip : dismissMode
         let gestureView = TouchForwardingView(frame: window.bounds)
         self.gestureView = gestureView
         switch self.dismissMode {
         case .tapAnywhere:
-            rootView?.addSubview(tooltipView)
-            rootView?.addSubview(gestureView)
+            hostView?.addSubview(tooltipView)
+            hostView?.addSubview(gestureView)
```

Comment:
I don't think the view of a view controller is optional, so we shouldn't need these `?` anymore.